src/pages/users/UsersPage.tsx

```diff
@@ -187,6 +231,7 @@ const UsersPage: FunctionComponent = () => {
         defaultValues: { user: '' }, //need default not undefined value for html input, else react error at runtime
     });
     const [open, setOpen] = useState(false);
+    const [show, setShow] = useState(false);
```

Comment:
showDeletionDialog ?
setShowDeletionDialog() can be used instead of handleDeletion()

Reply:
Changed.